packages/interfaces/src/protocols.ts

```diff
@@ -89,6 +89,17 @@ export type CreateNodeOptions = {
    */
   defaultBootstrap?: boolean;
 
+  /**
+   * Enable or disable specific discovery methods.
+   *
+   * @default { peerExchange: true, dns: true, localPeerCache: true }
+   */
+  discoveriesEnabled: {
```

Comment:
let's rename to simply `discovery`

```suggestion
  discovery: {
```